### PR TITLE
Update all references from Bitbucket to GitHub

### PR DESCRIPTION
**Pull request by vyasr**

### CODE REVIEW

File: doc/support.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ The continuous integration pipeline for the package will perform these checks as
 The git-flow development workflow
 ---------------------------------
 
-Prior to working on a patch, it is advisable to create an `issue <https://bitbucket.org/glotzer/signac-flow/issues?status=new&status=open>`_ that describes the problem or proposed feature.
+Prior to working on a patch, it is advisable to create an `issue <<https://github.com/glotzerlab/signac-flow/issues>>`_ that describes the problem or proposed feature.
```
Comment (csadorf) [inline doc/support.rst]: Looks like you got an superfluous `<` and `>` here.

File: doc/support.rst [hunk 2]
```diff
@@ -119,13 +116,13 @@ Then you can build the documentation from within the ``doc/`` directory as part
...
     Documentation as part of the package should be largely limited to the API.
-    More elaborate documentation on how to integrate **signac-flow** into a computational workflow should be documented as part of the `framework documentation <https://docs.signac.io>`_, which is maintained `here <https://bitbucket.org/glotzer/signac-docs>`_.
+    More elaborate documentation on how to integrate **signac-flow** into a computational workflow should be documented as part of the `framework documentation <https://docs.signac.io>`_, which is maintained `here <https://github.com/glotzerlab/signac-docs>`_.
 
 
 Updating the changelog
 ----------------------
 
-To update the changelog, add a one-line description to the `changelog.txt <https://bitbucket.org/glotzer/signac-flow/src/master/changelog.txt>`_ file within the ``next`` section.
+To update the changelog, add a one-line description to the `changelog.txt <https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt>`_ file within the ``next`` section.
```
Comment (csadorf) [inline doc/support.rst]: I've updated all changelogs links to point directly to the changelog on docs.signac.io/... .

Comment (vyasr) [reply doc/support.rst]: @csadorf I'll change this, but the link I see in [doc/support.rst](https://github.com/glotzerlab/signac/blob/develop/doc/support.rst) for signac on develop still links to GitHub, unless I'm looking in completely the wrong place.

**Comment by vyasr** [comment]
Regarding the flake8 issue, I was planning to just wait until the prior PR was merged. Now that that one is in develop I will merge that here, and hopefully that fixes any outstanding issues.

**Pull request merged**